src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3748,7 +3805,7 @@ private void CheckDisallowedNullAssignment(TypeWithState state, FlowAnalysisAnno
                         if (argument is BoundLocal local && local.DeclarationKind == BoundLocalDeclarationKind.WithInferredType)
                         {
                             var varType = worstCaseParameterWithState.ToTypeWithAnnotations();
-                            _variableTypes[local.LocalSymbol] = varType;
+                            SetLocalType(local, local.LocalSymbol, varType);
```

Comment:
>SetLocalType(local, local.LocalSymbol, varType); [](start = 28, length = 48)

Same comment as for VisitLocalDeclaration

Reply:
No longer applies as this code isn't changed.

---
In reply to: [327918526](https://github.com/dotnet/roslyn/pull/38769#discussion_r327918526) [](ancestors = 327918526)